### DISCUSSION TITLE
Updated repo is now throwing errors

### DISCUSSION THREAD

**Discussion by eurorackworld** [shanteacontrols/OpenDeck#292] [General]
I updated my repository and then 'making' a firmware failed. I did a clean install of the repo via 'git clone' and I still have the same errors when trying to execute the make command.

This is the output: 
---

~/OpenDeck$ make TARGET=BB-LH-V2
Generating CMake files
make[1]: *** /modules/core: No such file or directory.  Stop.
make[1]: *** /modules/core: No such file or directory.  Stop.
make[1]: *** /modules/core: No such file or directory.  Stop.
make[1]: *** /modules/core: No such file or directory.  Stop.
C_Make Error at src/firmware/board/CMakeLists.txt:5 (include):
  include could not find requested file:

    /home/###MYUSERNAME##/OpenDeck/src/generated/target/BB-LH-V2/CMakeLists.txt
Call Stack (most recent call first):
  CMakeLists.txt:37 (include)

CMake Error at src/firmware/board/CMakeLists.txt:19 (target_compile_definitions):
  Cannot specify compile definitions for target "mcu" which is not built by
  this project.
Call Stack (most recent call first):
  CMakeLists.txt:37 (include)

CMake Error at cmake/include/helpers.cmake:3 (add_subdirectory):
  add_subdirectory given source "/modules/LESSDB" which is not an existing
  directory.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:42 (add_module)
  CMakeLists.txt:39 (include)

CMake Error at cmake/include/helpers.cmake:8 (target_include_directories):
  Cannot specify include directories for target "lessdb-lib" which is not
  built by this project.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:43 (add_mcu_library)
  CMakeLists.txt:39 (include)

CMake Error at cmake/include/helpers.cmake:3 (add_subdirectory):
  add_subdirectory given source "/modules/SysExConf" which is not an existing
  directory.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:45 (add_module)
  CMakeLists.txt:39 (include)

CMake Error at cmake/include/helpers.cmake:8 (target_include_directories):
  Cannot specify include directories for target "sysexconf-lib" which is not
  built by this project.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:46 (add_mcu_library)
  CMakeLists.txt:39 (include)

CMake Error at cmake/include/helpers.cmake:3 (add_subdirectory):
  add_subdirectory given source "/modules/MIDI" which is not an existing
  directory.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:48 (add_module)
  CMakeLists.txt:39 (include)

CMake Error at cmake/include/helpers.cmake:8 (target_include_directories):
  Cannot specify include directories for target "midi-lib" which is not built
  by this project.
Call Stack (most recent call first):
  src/firmware/application/CMakeLists.txt:49 (add_mcu_library)
  CMakeLists.txt:39 (include)_

-- The C compiler identification is GNU 11.4.0
-- The CXX compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Configuring incomplete, errors occurred!
See also "/home/##MYUSERNAME##/OpenDeck/build/BB-LH-V2/release/CMakeFiles/CMakeOutput.log".
make: *** [Makefile:31: cmake_config] Error 1

----

Do I need to do something else?

**Comment by eurorackworld** [comment]
I don't see the modules directory anymore, also not on github. I see it was in the older version.

(also the get command is done very fast, while earlier it would take a while)

**Comment by paradajz** [comment]
The building procedure has changed - only Docker is now supported.
https://github.com/shanteacontrols/OpenDeck/wiki/Building-the-OpenDeck-firmware

**Reply by eurorackworld** [reply]
I just noticed , thanks for pointing me to that. 

(so far no luck though)

**Reply by eurorackworld** [reply]
no idea what I did, but after a lot of commands and installing docker and docker-compose I'm able to create the firmware again.
